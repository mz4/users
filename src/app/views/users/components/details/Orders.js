import { Subtitle } from '../../../../library/index';
import styles from './Orders.module.scss';

const Orders = () => {
  return (
    <>
      <div className={styles.box}>
        <Subtitle text="Experience" />
        <div className={styles.orderValue}>5 years</div>
      </div>
      <div className={styles.box}>
        <Subtitle text="Level" />
        <div className={styles.orderValue}>Senior</div>
      </div>
      <div className={styles.box}>
        <Subtitle text="Projects" />
        <div className={styles.orderValue}>5</div>
      </div>
    </>
  );
};

export default Orders;
