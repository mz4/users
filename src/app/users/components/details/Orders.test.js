import renderer from 'react-test-renderer';
import Orders from './Orders';

describe('Render component', () => {
  it('Create a snapshot', () => {
    const orders = <Orders />;
    const component = renderer.create(orders);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
