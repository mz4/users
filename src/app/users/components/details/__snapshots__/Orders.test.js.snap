// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render component Create a snapshot 1`] = `
Array [
  <div
    className="box"
  >
    <div
      className="pageSubTitle"
    >
      <span>
        Total Orders Today
      </span>
    </div>
    <div
      className="orderValue"
    >
      12.450 $
    </div>
  </div>,
  <div
    className="box"
  >
    <div
      className="pageSubTitle"
    >
      <span>
        Total Clients Today
      </span>
    </div>
    <div
      className="orderValue"
    >
      25
    </div>
  </div>,
  <div
    className="box"
  >
    <div
      className="pageSubTitle"
    >
      <span>
        Revenue Day Ratio
      </span>
    </div>
    <div
      className="orderValue"
    >
      25%
    </div>
  </div>,
]
`;
